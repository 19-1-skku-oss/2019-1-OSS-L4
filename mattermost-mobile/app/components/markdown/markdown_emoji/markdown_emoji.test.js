// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import React from 'react';
import {shallow} from 'enzyme';

import Preferences from 'mattermost-redux/constants/preferences';

import MarkdownEmoji from './markdown_emoji';

describe('MarkdownEmoji', () => {
    const baseProps = {
        baseTextStyle: {color: '#3d3c40', fontSize: 15, lineHeight: 20},
        isEdited: false,
        shouldRenderJumboEmoji: true,
        theme: Preferences.THEMES.default,
        value: ':smile:',
    };

    test('should match snapshot', () => {
        const wrapper = shallow(
            <MarkdownEmoji {...baseProps}/>
        );

        expect(wrapper.getElement()).toMatchSnapshot();
    });
});
