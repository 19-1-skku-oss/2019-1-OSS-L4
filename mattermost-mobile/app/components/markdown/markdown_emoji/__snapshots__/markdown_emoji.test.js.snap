// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MarkdownEmoji should match snapshot 1`] = `
<Unknown
  context={Array []}
  first={true}
  last={true}
  literal={null}
  nodeKey="5"
>
  <Unknown
    context={
      Array [
        "paragraph",
      ]
    }
    emojiName="smile"
    literal=":smile:"
    nodeKey="4"
  >
    <Unknown
      context={
        Array [
          "paragraph",
          "emoji",
        ]
      }
      literal=":smile:"
      nodeKey="3"
    />
  </Unknown>
</Unknown>
`;
